core/textfield.go

```diff
@@ -149,6 +149,9 @@ type TextField struct { //core:embedder
 	// selectMode is whether to select text as the cursor moves.
 	selectMode bool
 
+	// selectModeShift is whether selectmode was turned on because of the shift key.
+	selectModeShift bool
```

Comment:
We don't also need this for text editor?

Reply:
nope -- it works fine as-is -- has more sophisticated logic.